server/events.go

```diff
@@ -920,6 +934,9 @@ func (s *Server) accountConnectEvent(c *client) {
 	}
 
 	m := ConnectEventMsg{
+		Schema: ConnectEventMsgSchema,
+		ID:     nuid.Next(),
```

Comment:
nuid.Next() uses a the global struct and will have a "global lock". I wonder if we should create an instance of Nuid that we store in the server's event structure and get it to call Next() on it under the server lock above. No big deal but just reducing possible contention if we start to have lots of events.

Reply:
I was wondering about that when I did the JS ones, there the volume of messages isnt huge so parked investigating, here I think you're right we should do this.